pwa/src/templates/componentDetail/ComponentsDetailTemplate.tsx

```diff
@@ -43,6 +43,7 @@ import { DependenciesTemplate } from "../templateParts/dependenciesTemplates/Com
 import { FiltersContext } from "../../context/filters";
 import { ComponentCardsAccordionTemplate } from "../templateParts/componentCardsAccordion/ComponentCardsAccordionTemplate";
 import { DownloadTemplate } from "../templateParts/download/DownloadTemplate";
+import { TEMPORARY_ORGANIZATIONS } from "../../data/organizations";
```

Comment:
Using temporary Organizations is good for now but we will have to replace them in the future